### DISCUSSION TITLE
On leave until end of June

### DISCUSSION THREAD

**Discussion by ochadenas** [ochadenas/cpudefense#156] [Announcements]
Hi all,

I will not have the possibility to work on the game in the next weeks. The next update will not be before July, so please be patient! New features will have to wait, and I hope that there are not _that_ many bugs in the current version.

**Comment by ochadenas** [comment]
Hi everyone,

I'm back, and hopefully will have some time to spend on development. I'll try to implement resistors next.

**Reply by Mortis03** [reply]
Great!  

Still playing here at stage 419 on endless
